services/galley/src/Galley/API/MLS/Message.hs

```diff
@@ -768,10 +781,14 @@ getMLSClients ::
   Local x ->
   Qualified UserId ->
   SignatureSchemeTag ->
-  Sem r (Set ClientId)
+  Sem r (Set ClientId, Set ClientId)
```

Comment:
I keep on being confused and worried about pair types with the same type in each component, i.e., `(a, a)`. While reviewing this PR I already went a few times back and forth to check what is assigned to what component of the pair, and I find it to be using time, while it shouldn't. Do you have a suggestion on how to make them separate types so to avoid confusing the components easily, yet such that is still easy to use?

Reply:
It's two subsets, ordered by inclusion. I thought that would be the most intuitive way to represent this. But you're probably right, we need to create a new type.

Reply:
I turned it into a single set of `ClientInfo`, where `ClientInfo` is essentially `(ClientId, Bool)`.